src/cmdlib.sh

```diff
@@ -126,16 +126,18 @@ print(disk_ignition_version('${path}}'))"
 }
 
 # Picks between yaml or json based on which version exists. If neither exist,
-# prefers YAML.
+# prefers ${default}, or YAML if not specified.
```

Comment:
Is this correct? It looks like:

If we find both, error.
If we find json, use json.
If we find yaml, use yaml.
If we don't find either, then use the default.

Reply:
I tweaked the wording a bit! But yes, that's essentially it with an extra step:

"If the default is not given, then use yaml"